NitroxModel/Helper/Unity/GameObjectHelper.cs

```diff
@@ -28,6 +28,22 @@ public static class GameObjectHelper
             return component;
         }
 
+        public static T[] RequireComponentsInChildren<T>(this GameObject o) where T : class
+        {
+            T[] component = o.GetComponentsInChildren<T>();
+            Validate.NotNull<T[]>(component, o.name + " did not have a component of type " + typeof(T) + " in its children");
```

Comment:
`GetComponentsInChildren` will never return `null`, it returns an empty list when the component can't be found.

Also, the type specifier in `NotNull` is not needed - the compiler derives it from the type (agreed though, the othe functions do it as well).